.github/workflows/translation-updates-to-main-repo.yml

```diff
@@ -0,0 +1,144 @@
+            languagelist=${{ github.event.inputs.languages }}
+          else
+            languagelist=$(cat _config.yml | grep "languages: \[.*\]" )
+            languagelist=${languagelist//[}
+            languagelist=${languagelist//]}
+            echo $languagelist
+            languagelist=${languagelist//'languages: '}
+          fi
+          languagelist=${languagelist//,}
+          languagelist=${languagelist//;}
+          # Exclude English (source) from list of languages to update
+          languagelist=${languagelist//'en'}
+          # Make variable consistent between steps
+          echo "languages=$languagelist" >> $GITHUB_ENV
+      - name: Print tree
+        run: |
+          sudo apt-get install tree
+          tree -L 2
+      - name: Print languages
+        run: |
+          languages=(${{ env.languages }})
+          for language in "${languages[@]}"
+          do echo $language
+          done
+      - name: Make Contribution pages
+        run: |
+          cd _i18n/en/contribute
+          languages=(${{ env.languages }})
+          for language in "${languages[@]}"
+          do mkdir -p ../../$language/contribute
+          done
+          for file in *
+          do
+            for language in "${languages[@]}"
+            do po2md $file --po-files ../../../translation-files/contribute-pages/site-contribute_$language.po --save ../../$language/contribute/$file
+            git add ../../$language/contribute/$file
+            done
+          done
+      - name: Make Documentation pages
+        run: |
+          # top-level md files
+          cd _i18n/en/documentation
+          languages=(${{ env.languages }})
+          for language in "${languages[@]}"
+          do mkdir -p ../../$language/documentation
+          done
+          for file in *.md
+          do
+            for language in "${languages[@]}"
+            do po2md $file --po-files ../../../translation-files/documentation-pages/site-documentation_$language.po --save ../../$language/documentation/$file
+            git add ../../$language/documentation/$file
+            done
+          done
+          # sub-level files
+          documentation_groups=(automation general getting-started playback podcasters-hosters queue subscriptions)
+          for category in "${documentation_groups[@]}"
+          do
+            cd $category
+            languages=(${{ env.languages }})
+            for language in "${languages[@]}"
+            do mkdir -p ../../../$language/documentation/$category
+            done
+            for file in *
+            do
+              for language in "${languages[@]}"
+              do
+                po2md $file --po-files ../../../../translation-files/documentation-pages/site-documentation_$language.po --save ../../../$language/documentation/$category/$file
+              git add ../../../$language/documentation/$category/$file
+              done
+            done
+            cd ../
+          done
```

Comment:
I would move that to a bash script. That's so long and having it inside the yaml file makes it hard to read/maintain